src/main/java/formflow/library/FileController.java

```diff
@@ -94,9 +94,10 @@ public ResponseEntity<?> upload(
     log.info("POST upload (url: {}): flow: {}, inputName: {}", request.getRequestURI().toLowerCase(), flow, inputName);
     try {
       if (!doesFlowExist(flow)) {
-        throwNotFoundError(flow, null, String.format("Could not find flow with name %s in your application's flow configuration.", flow));
+        throwNotFoundError(flow, null,
```

Comment:
Is your styles/linter adding a line break?

Reply:
Yep, this was the linter.